### PR TITLE
Add email OTP to auth methods policy config

### PR DESCRIPTION
**Pull request by mmcla**
Adding email OTP to the authentication methods configuration

### CODE REVIEW

File: api-reference/beta/api/emailauthenticationmethodconfiguration-delete.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|Policy.ReadWrite.AuthenticationMethod|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|Not supported.|
+
+For delegated scenarios, the administrator needs one of the following [roles](/azure/active-directory/users-groups-roles/directory-assign-admin-roles#available-roles):
```
Comment (andres-canello) [inline api-reference/beta/api/emailauthenticationmethodconfiguration-delete.md:27]: Only Global Admins can do this right now. I have another PR waiting to make these changes to the live docs. The working I used is: 

For delegated scenarios, the administrator needs the following [role](/azure/active-directory/users-groups-roles/directory-assign-admin-roles#available-roles):

* Global admin

File: api-reference/beta/api/emailauthenticationmethodconfiguration-get.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|Policy.ReadWrite.AuthenticationMethod|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|Not supported.|
+
+For delegated scenarios the administrator needs one of the following [roles](/azure/active-directory/users-groups-roles/directory-assign-admin-roles#available-roles):
```
Comment (andres-canello) [inline api-reference/beta/api/emailauthenticationmethodconfiguration-get.md:28]: Same comment about GA.

File: api-reference/beta/api/emailauthenticationmethodconfiguration-update.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|Policy.ReadWrite.AuthenticationMethod|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|Not supported.|
+
+For delegated scenarios, the administrator needs one of the following [roles](/azure/active-directory/users-groups-roles/directory-assign-admin-roles#available-roles):
```
Comment (andres-canello) [inline api-reference/beta/api/emailauthenticationmethodconfiguration-update.md:27]: Same comment about GA.

File: api-reference/beta/resources/emailauthenticationmethodconfiguration.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+|[Update emailAuthenticationMethodConfiguration](../api/emailauthenticationmethodconfiguration-update.md)|[emailAuthenticationMethodConfiguration](../resources/emailauthenticationmethodconfiguration.md)|Update the properties of an [emailAuthenticationMethodConfiguration](../resources/emailauthenticationmethodconfiguration.md) object.|
+|[Delete emailAuthenticationMethodConfiguration](../api/emailauthenticationmethodconfiguration-delete.md)|None|Deletes an [emailAuthenticationMethodConfiguration](../resources/emailauthenticationmethodconfiguration.md) object.|
+
+## Properties
+
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|The authentication method policy identifier. Inherited from [authenticationMethodConfiguration](../resources/authenticationmethodconfiguration.md)|
+|state|authenticationMethodState|Whether this authentication method is enabled or not. Possible values are: `enabled`, `disabled`.|
+|allowExternalIdToUseEmailOtp|externalEmailOtpState|Determines whether email OTP is usable by external users for authentication. Possible values are: `default`, `enabled`, `disabled`, `unknownFutureValue`.|
```
Comment (andres-canello) [inline api-reference/beta/resources/emailauthenticationmethodconfiguration.md:32]: I think we need to explain what the "default" state means.

**Review by andres-canello — COMMENTED** [review]
Left a few notes.  I think you are missing the entry in the changelog.

File: api-reference/beta/api/emailauthenticationmethodconfiguration-get.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+
+```http
+GET https://graph.microsoft.com/beta/policies/authenticationMethodsPolicy/authenticationMethodConfiguration/email
```
Comment (Lauragra) [inline api-reference/beta/api/emailauthenticationmethodconfiguration-get.md:43]: In this section we don't show the full URL, we do that in the example. Just show the syntax here.

File: api-reference/beta/api/emailauthenticationmethodconfiguration-get.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+doc_type: "apiPageType"
+---
+
+# Get emailAuthenticationMethodConfiguration
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Read the properties and relationships of the [emailAuthenticationMethodConfiguration](../resources/emailauthenticationmethodconfiguration.md) object, which represents the email OTP [authentication method policy](../resources/authenticationmethodspolicies-overview.md) for the Azure Active Directory (Azure AD) tenant.
```
Comment (Lauragra) [inline api-reference/beta/api/emailauthenticationmethodconfiguration-get.md:16]: Is OTP an acronym that will be familiar to readers? Generally we spell out acronyms on first mention.

File: api-reference/beta/resources/emailauthenticationmethodconfiguration.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents this tenant's email OTP authentication methods policy. Authentication methods policies define configuration settings and users or groups who are enabled to use the authentication method. Email OTP may be used by the tenant's cloud-native users for self-service password reset, or by external users for authentication in some circumstances.
+
+## Methods
+
+|Method|Return type|Description|
+|:---|:---|:---|
+|[Get emailAuthenticationMethodConfiguration](../api/emailauthenticationmethodconfiguration-get.md)|[emailAuthenticationMethodConfiguration](../resources/emailauthenticationmethodconfiguration.md)|Read the properties and relationships of an [emailAuthenticationMethodConfiguration](../resources/emailauthenticationmethodconfiguration.md) object.|
```
Comment (Lauragra) [inline api-reference/beta/resources/emailauthenticationmethodconfiguration.md:22]: Remove the links from the Description column.

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please incorporate comments and please add an update to the changelog.md file and the DirectoryServices.json changelog file for the new APIs.

File: api-reference/beta/api/emailauthenticationmethodconfiguration-get.md (hunk 5 above)
Comment (mmcla) [reply api-reference/beta/api/emailauthenticationmethodconfiguration-get.md:43]: Ok, I'm pretty sure these were added by the Graph team's autogenerator tool, probably worth checking to see if that's something they're introducing.

File: api-reference/beta/api/emailauthenticationmethodconfiguration-update.md (hunk 3 above)
Comment (mmcla) [reply api-reference/beta/api/emailauthenticationmethodconfiguration-update.md:27]: Fixed

File: api-reference/beta/resources/emailauthenticationmethodconfiguration.md (hunk 7 above)
Comment (mmcla) [reply api-reference/beta/resources/emailauthenticationmethodconfiguration.md:22]: Fixed

**Review by davidmu1 — APPROVED** [review]
Other than the current requested changes and the API Doctor errors, it looks good to me.

**Comment by mmcla** [comment]
> Please incorporate comments and please add an update to the changelog.md file and the DirectoryServices.json changelog file for the new APIs.

@Lauragra I added the changelog entries and updated all of the changes you requested.

Everything is done except for one suggested change from Andres, which I'm waiting on a sentence from a colleague.

**Review by Lauragra — APPROVED** [review]

**Review by andrewchoi5 — APPROVED** [review]

**Pull request merged**